src/components/FiltersDrawer/index.tsx

```diff
@@ -0,0 +1,111 @@
+import { useState } from "react";
+import styled from "@emotion/styled";
+import Badge, { Variant } from "@leafygreen-ui/badge";
+import { SideNav, SideNavGroup } from "@leafygreen-ui/side-nav";
+import { Body } from "@leafygreen-ui/typography";
+import Icon from "components/Icon";
+import { QueryParams } from "constants/queryParams";
+import { size, zIndex } from "constants/tokens";
+import { useQueryParam } from "hooks/useQueryParam";
+import Filter from "./Filter";
+
+interface FiltersDrawerProps {
+  ["data-cy"]?: string;
+}
+
+const FiltersDrawer: React.FC<FiltersDrawerProps> = ({ "data-cy": dataCy }) => {
+  const [collapsed, setCollapsed] = useState(true);
+
+  const [filters, setFilters] = useQueryParam<string[]>(
+    QueryParams.Filters,
+    []
+  );
+
+  const deleteFilter = (filterName: string) => {
+    const newFilters = filters.filter((f) => f !== filterName);
+    setFilters(newFilters);
+  };
+
+  const editFilter = (oldFilter: string, newFilter: string) => {
+    // Duplicate filters are not allowed.
+    if (filters.includes(newFilter)) {
+      return;
+    }
+    const newFilters = [...filters];
+    const idxToReplace = newFilters.indexOf(oldFilter);
+    newFilters[idxToReplace] = newFilter;
+    setFilters(newFilters);
+  };
+
+  return (
+    <StyledSideNav
+      aria-label="Filters Side Nav"
+      collapsed={collapsed}
+      data-cy={dataCy}
+      setCollapsed={setCollapsed}
+      widthOverride={250}
+    >
+      <PaddedContainer>
+        <StyledSideNavGroup
+          glyph={<Icon glyph="Filter" />}
```

Comment:
It'd be sweet if this icon appeared inactive (i.e. grey) when 0 filters are applied... but I'm looking at the LG component right now and it doesn't appear so easy 😿

Reply:
when there's a will, there's a way 😁 if you would like me to tweak the colors, let me know!